### PR TITLE
Fixed some pep8 violations

### PR DESCRIPTION
**Pull request by iamgroot42**
Ref : https://github.com/tardis-sn/tardis/issues/427

### CODE REVIEW

**Comment by iamgroot42** [comment]
Can somebody review this PR? I only fixed some PEP8 violations, yet the build doesn't pass.

File: tardis/io/tests/test_model_reader.py [hunk 1]
```diff
@@ -5,31 +5,35 @@
 import pytest
 
 import tardis
-from tardis.io.model_reader import (read_artis_density, 
-read_simple_ascii_abundances)
+from tardis.io.model_reader import read_artis_density, \
```
Comment (yeganer) [inline tardis/io/tests/test_model_reader.py]: I like the old way better. In my opinion `\` should not be used for linebreaks in python.

Comment (iamgroot42) [inline tardis/io/tests/test_model_reader.py]: The only alternative is to repeat the _from_ part, but I don't like that either ; feels a bit redundant.

Comment (yeganer) [inline tardis/io/tests/test_model_reader.py]: the preferred method is:
```
from tardis.io.model_reader import (
    read_artis_density, 
    read_simple_ascii_abundances)
```

Comment (iamgroot42) [inline tardis/io/tests/test_model_reader.py]: Fair enough. I'll make the appropriate changes

Comment (wkerzendorf) [inline tardis/io/tests/test_model_reader.py]: Sorry we do not accept PEP8 fixes at this time.
On Fri, 4 Mar 2016 at 09:05, Anshuman Suri <notifications@github.com> wrote:

> In tardis/io/tests/test_model_reader.py
> <https://github.com/tardis-sn/tardis/pull/504#discussion_r55000528>:
>
> > @@ -5,31 +5,35 @@
> >  import pytest
> >
> >  import tardis
> > -from tardis.io.model_reader import (read_artis_density,
> > -read_simple_ascii_abundances)
> > +from tardis.io.model_reader import read_artis_density, \
>
> Fair enough. I'll make the appropriate changes
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/tardis-sn/tardis/pull/504/files#r55000528>.
>
-- 
Dr. Wolfgang E Kerzendorf
ESO Fellow
European Southern Observatory
Karl-Schwarzschild Str. 2, 85748
Garching bei Muenchen, Germany

**Pull request closed without merging**